Listener/ResponseListener.php

```diff
@@ -85,18 +85,17 @@ public function onKernelResponse(FilterResponseEvent $event): void
                 // We can only instrument HTML responses
                 if ('text/html' === \substr($response->headers->get('Content-Type'), 0, 9)) {
                     $responseContent = $response->getContent();
+                    $response->setContent(''); // free the memory
 
                     if (null === $this->newRelicTwigExtension || false === $this->newRelicTwigExtension->isHeaderCalled()) {
-                        $responseContent = \preg_replace('/<\s*head\s*>/', '$0'.$this->interactor->getBrowserTimingHeader(), $responseContent);
+                        $responseContent = \preg_replace('|<head>|i', '$0'.$this->interactor->getBrowserTimingHeader(), $responseContent);
```

Comment:
Im fine with changing the regex here (since we do 2.0). If there is causing any issue for someone then they will provide a better test.